Detectors/TRD/calibration/src/TrackBasedCalib.cxx

```diff
@@ -77,6 +90,78 @@ void TrackBasedCalib::calculateAngResHistos()
   // mAngResHistos.print();
 }
 
+int TrackBasedCalib::filldEdx(gsl::span<const TrackTRD>& tracks, bool isTPCTRD)
+{
+  auto& params = TRDCalibParams::Instance();
```

Comment:
How about moving this to a class member and removing all the calls to Instance()?